docs/web-ui.md

```diff
@@ -49,6 +49,50 @@ sizes and using executors for all partitions in an RDD or DataFrame.
 The Environment tab displays the values for the different environment and configuration variables,
 including JVM, Spark, and system properties.
 
+<p style="text-align: center;">
+  <img src="img/webui-env-tab.png"
+       title="Env tab"
+       alt="Env tab"
+       width="100%" />
+  <!-- Images are downsized intentionally to improve quality on retina displays -->
+</p>
+
+We can see that this environment page has five parts, it is a useful place to check whether your properties have
```

Comment:
I might use a different paragraph for each of the five sections you mention.
"it is" should start a new sentence.
You can remove wording like "We can see that"